device/globals/player.gd

```diff
@@ -244,7 +244,9 @@ func _process(time):
 		else:
 			next = walk_path[path_ofs]
 
-		var dist = speed * time * last_scale.x * last_scale.x
+		var dist = speed * time * last_scale.x * last_scale.x * terrain.player_speed_multiplier
```

Comment:
Can `last_scale.x * last_scale.x` be written as `last_scale.x ** 2` or `last_scale.x ^ 2`? Should take precendence over multiplication and is more concise.

Reply:
Suppose it could be. Not my original code but I can push that in.

Reply:
Actually it can not. I either get a syntax error on `last_scale.x ** 2` or unsupported operand for `last_scale.x ^2` so we're stuck with what we got.

Reply:
Yeah, it's `pow(last_scale, 2)`:

http://docs.godotengine.org/en/3.0/classes/class_@gdscript.html?highlight=pow#class-gdscript-pow